support/spray/src/main/scala/sjsonnew/support/spray/Converter.scala

```diff
@@ -76,8 +76,11 @@ object Converter extends SupportConverter[JsValue] {
       value match {
         case x: JsObject =>
           val fields = x.fields
-          val names = (x.fields map { case (k, v) => k }).toVector
-          (fields, names)
+          val vectorBuilder = Vector.newBuilder[String]
```

Comment:
The API here should change. There is no need to have a `Vector`, at most it should be an `Array` since we're only checking indexes in the use sites. Also, we don't add things to this array, so fixed size is good.